test/unit/sqs/sqs-consumer-spec.js

```diff
@@ -368,6 +368,34 @@ describe('SqsConsumer', () => {
           });
       });
     });
+
+    context('when there are no messages to consume', () => {
+
+      before(() => {
+        consumer = new SqsConsumer({sqs: sqs}, msgBody => Promise.resolve(), {sqs: config});
+      });
+
+      it('resolves the original object: undefined', () => {
+        consumer._scheduledConsuming(undefined)
```

Comment:
missing return statement (If promis does not resolve for any reason, test will timeout instead of showing error)